pkgs/development/python-modules/rst2ansi/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, docutils
+}:
+
+buildPythonPackage rec {
+  pname = "rst2ansi";
+  version = "0.1.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0vzy6gd60l79ff750scl0sz48r1laalkl6md6dwzah4dcadgn5qv";
+  };
+
+  propagatedBuildInputs = [ docutils ];
+
+  doCheck = true;
+
+  meta = with lib; {
+    homepage = "https://github.com/Snaipe/python-rst-to-ansi";
+    license = licenses.mit;
+    description = "A rst converter to ansi-decorated console output";
```

Comment:
Please add yourself as a maintainer when adding a new package.

https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-new-packages

Reply:
I saw a lot of other python packages didn't have maintainers, so I didn't realise this was the same as top level packages.